Jellyfin.Server/Extensions/ApiServiceCollectionExtensions.cs

```diff
@@ -344,13 +344,20 @@ internal static void AddProxyAddresses(NetworkConfiguration config, string[] all
         {
             for (var i = 0; i < allowedProxies.Length; i++)
             {
-                if (IPNetAddress.TryParse(allowedProxies[i], out var addr))
+                if (IPAddress.TryParse(allowedProxies[i], out var addr))
                 {
-                    AddIpAddress(config, options, addr.Address, addr.PrefixLength);
+                    AddIpAddress(config, options, addr, addr.AddressFamily == AddressFamily.InterNetwork ? 32 : 128);
                 }
-                else if (IPHost.TryParse(allowedProxies[i], out var host))
+                else if (NetworkExtensions.TryParseSubnets(new[] { allowedProxies[i] }, out var subnets))
```

Comment:
Would be better with a non-array overload instead of creating a one item array. But if this is only done during startup it's not super important